### PR TITLE
#134: Add defensive programming to fix error of DateTime

### PR DESCRIPTION
**Pull request by edutrul**
Related issue: https://github.com/Lullabot/media_mpx/issues/134

Add defensive programming to fix an error of DateTime

### CODE REVIEW

**Comment by edutrul** [comment]
Is this a legit error? @deviantintegral 

image

**Comment by edutrul** [comment]
Great that's not displaying anymore

File: src/Access/MediaAvailableAccess.php [hunk 1]
```diff
@@ -206,7 +206,8 @@ protected function getStubMediaObject(MediaInterface $media): MpxMedia {
    */
   private function getDateTime(MediaInterface $media, string $field_name): DateTimeFormatInterface {
     $fieldItemList = $media->get($field_name);
-    $date = !$fieldItemList->isEmpty() ? new ConcreteDateTime(\DateTime::createFromFormat(DateTimeItemInterface::DATETIME_STORAGE_FORMAT, $fieldItemList->value)) : new NullDateTime();
+    $date_time = \DateTime::createFromFormat(DateTimeItemInterface::DATETIME_STORAGE_FORMAT, $fieldItemList->value);
```
Comment (deviantintegral) [inline src/Access/MediaAvailableAccess.php:209]: What is $fieldItemList->value when it fails? An empty string? I'm surprised the isEmpty() check isn't covering this.

Comment (edutrul) [reply src/Access/MediaAvailableAccess.php:209]: I debugged and got an integer value which is a valid timestamp value

**Comment by juampynr** [comment]
conflicts

**Comment by edutrul** [comment]
Great now it is all green 💚  @juampynr

**Review by germanmontesm — APPROVED** [review]

**Pull request merged**